Public/Functions/split/Update-MyWindowsImage.ps1

```diff
@@ -91,7 +91,7 @@ function Update-MyWindowsImage {
             #=================================================
             $global:GetWSUSXML = Get-WSUSXML -Catalog Windows -Silent | Sort-Object UpdateGroup -Descending
 
-            if ($global:GetRegCurrentVersion.ReleaseId -gt 0) {
+            if ($global:GetRegCurrentVersion.DisplayVersion -gt 0) {
```

Comment:
DisplayVersion may not always be an int.  If you are looking for the minor version (revision) check out GetRegCurrentVersion.UBR.